addbook.php

```diff
@@ -0,0 +1,56 @@
+<?php
+require 'functions.php';
+
+$db = startDb('books');
+
+if (!empty($_POST)) {
```

Comment:
in the same way as the login script, its best practise to do this in a different file. dont make the file display the form AND insert the data, have 1 file for each